src/regexps.ts

```diff
@@ -28,12 +28,12 @@ export const anyLine = XRegExp('^ (?<line> .* ) $', 'nx');
 
 
 // Re-builds the 🍂 directive based on a different leading character
-export function redoLeafDirective(char) {
+export function redoLeafDirective(char: string): any {
 	return leafDirective = XRegExp('  \\s* ' + char + ' (?<directive> \\S+ ) (\\s+ (?<content> [^;\\n]+ )){0,1} ', 'gnx');
 }
 
 // Parses a 🍂 directive, init'd at redoLeafDirective()
-export const leafDirective = redoLeafDirective('🍂');
```

Comment:
Another bug TypeScript caught: `const` variables can not be re-assigned to, which the above `redoLeafDirective` was trying to do. Not sure why the problem wouldn't have been caught before (maybe Rollup converting everything to `var` and not checking re-assignment; that would explain it)?